tools/src/test/resources/1.5/valid-vulnerability-1.5.xml

```diff
@@ -80,14 +80,16 @@
                 </individuals>
             </credits>
             <tools>
-                <tool>
-                    <vendor>Snyk</vendor>
-                    <name>Snyk CLI (Linux)</name>
-                    <version>1.729.0</version>
-                    <hashes>
-                        <hash alg="SHA-256">2eaf8c62831a1658c95d41fdc683cd177c147733c64a93e59cb2362829e45b7d</hash>
-                    </hashes>
-                </tool>
+                <components>
+                    <component type="application">
+                        <group>Snyk</group>
+                        <name>Snyk CLI (Linux)</name>
+                        <version>1.729.0</version>
+                        <hashes>
+                            <hash alg="SHA-256">2eaf8c62831a1658c95d41fdc683cd177c147733c64a93e59cb2362829e45b7d</hash>
+                        </hashes>
+                    </component>
+                </components>
```

Comment:
could you also add `<services>`? 
So we would have a complete example.

Reply:
Good catch. Added in https://github.com/CycloneDX/specification/pull/198/commits/bba0493f9c44faa67e436eee1fe0685c152ad6d7